Adding more query examples (#103387)
Azure/azure-content: articles/azure-functions/analyze-telemetry-data.md

The runtime provides the `customDimensions.LogLevel` and `customDimensions.Category` fields. You can provide additional fields in logs that you write in your function code. For an example in C#, see [Structured logging](functions-dotnet-class-library.md#structured-logging) in the .NET class library developer guide.

> ## Query Function Invocation
> 
> Every Function invocation is assigned a unique id. InvocationId is included in the custom dimention and can be used to coorelate all the logs from a particular function invocation.

Reviewer: Just some minor edits. Note that Azure docs use `Sentence case` and not `Title Case` for headings. 
```suggestion
## Query function invocations

Every function invocation is assigned a unique ID. `InvocationId` is included in the custom dimension and can be used to correlate all the logs from a particular function execution.
```

```kusto
traces
| project customDimensions["InvocationId"], message
```

## Telemetry correlation

> Logs from different functions may be correlated using operation_Id. Use the following query to get all the telemetry for a logical operation.

Reviewer: ```suggestion
Logs from different functions can be correlated using `operation_Id`. Use the following query to return all the logs for a specific logical operation.
```

```kusto
traces
| where operation_Id == '45fa5c4f8097239efe14a2388f8b4e29'
| project timestamp, customDimensions["InvocationId"], message
| order by timestamp
```

> ## Sampling Percentage
> 
> Sampling configuration can be used to reduce the volume of telemetry. Use the following query to determine if sampling is operational or not. If you see that RetainedPercentage for any type is less than 100, then that type of telemetry is being sampled.

Reviewer: ```suggestion
## Sampling percentage

Sampling configuration can be used to reduce the volume of telemetry. Use the following query to determine if sampling is operational or not. If you see that `RetainedPercentage` for any type is less than 100, then that type of telemetry is being sampled.
```
